server/work-queue/queue.js

```diff
@@ -0,0 +1,108 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+'use strict';
+
+var nr = require('node-resque');
```

Comment:
for right now, most of the contents here are pasted straight out of the node-resque readme. "works on my machine" ;-P